<launch>
  <!-- 可调参数 -->
  <arg name="twist_cmd_topic" default="/cmd_vel" />
  <arg name="ctrl_cmd_topic"  default="/ctrl_cmd" />

  <!-- 如果你还有别的节点（比如 YHS CAN 驱动），可以在这里一并 include -->

  <!-- carlike 运动学桥接：/cmd_vel -> /ctrl_cmd -->
  <node pkg="sentry_kinematics_bridge"
        type="twist_to_yhs_ctrl.py"
        name="twist_to_yhs_ctrl_bridge"
        output="screen">

    <param name="twist_cmd_topic" value="$(arg twist_cmd_topic)" />
    <param name="ctrl_cmd_topic"  value="$(arg ctrl_cmd_topic)" />
  </node>

</launch>
